config/canonical.json

```diff
@@ -20404,6 +20421,8 @@
     "count": 145,
     "tags": {
       "brand": "Jules",
+      "brand:wikidata": "Q25351738",
+      "brand:wikipedia": "en:Joules (clothing)",
```

Comment:
I looked at this one earlier. I don't think this is the right wikipedia article. That's a UK clothing store right above this entry and most of the Jules stores are in France. Different place.

Reply:
Good catch. Your probably right.